bin/main.js

```diff
@@ -18,13 +18,10 @@ export async function parseArgv(argv) {
       
     Storybook options
       --build-script-name [name], -b  The npm script that builds your Storybook [build-storybook]
-      --script-name [name], -s  The npm script that starts your Storybook [storybook]
-      --exec <command>, -e  Alternatively, a full command to run to start your storybook
-      --do-not-start, -S  Don't attempt to start or build; use if your Storybook is already running
+      --storybook-build-dir, -d  <dirname>  Provide a directory with your built storybook; use if you've already built your storybook
       
       --storybook-port <port>, -p  What port is your Storybook running on (auto detected from -s, if set)?'
```

Comment:
I guess we should drop the `--storybook-port` and https ones too?

Reply:
I agree